scripts/packing_similarity_dendrogram/packing_similarity_dendogram.py

```diff
@@ -395,9 +403,16 @@ def main(input_file, matrix_file, n_ps_mols, output_ps_results, conf_threshold,
     y = np.arange(0, structure_size + 1, 1)
 
     plot = plt.pcolor(x, y, matrix, cmap=plt.get_cmap('rainbow', (n_ps_mols - 1)), vmin=1, vmax=n_ps_mols)
-    plt.xticks(np.arange(0, structure_size + 1, 5) - 0.5, np.arange(0, structure_size + 1, 5))
-    plt.yticks(np.arange(0, structure_size + 1, 5) - 0.5, np.arange(0, structure_size + 1, 5))
-    cb = plt.colorbar(plot, ticks=range(1, 16))
+    if structure_size < 10:
+        plt.xticks([x -0.5 for x in list(range(1, structure_size+1))], list(range(1, structure_size+1)))
```

Comment:
How come the range is wrapped in a list? Does the generate throw some strange things?

Reply:
A change by Jonas: axis tick marks start running together if there's loads of structures and the tick iteration is 1, but tick iteration of 5 is too sparse for small sets (Jonas was using it for the lowest 9 or so CSP structures, I believe). I've just updated it, so both loops use the original np.arange style